src/unix/fsevents.c

```diff
@@ -642,15 +642,15 @@ static int uv__fsevents_loop_init(uv_loop_t* loop) {
   }
 
   /* In the unlikely event that pthread_attr_init() fails, create the thread
-   * with the default stack size. We'll use a little more address space but
-   * that in itself is not a fatal error.
+   * with the default stack size. We'll likely use less address space but that
+   * in itself is not a fatal error.
    */
   attr = &attr_storage;
   if (pthread_attr_init(attr))
     attr = NULL;
```

Comment:
There's an argument for just calling `abort()` now like we do here:

https://github.com/libuv/libuv/blob/285a5ea819035ff777b8b7c6a367f3f5b55d8809/src/unix/thread.c#L251-L252

Reply:
I'm okay with that. `pthread_attr_init()` on macos is quite literally infallible.